src/index.ts

```diff
@@ -34,21 +34,28 @@ export default {
     const res = await fetch(`${Config.STATUS_URL}/api/v2/incidents.json`)
     const json = await res.json<IncidentResponse>();
 
-    await Promise.all(json.incidents.map(async incident => {
-      const kv = await env.KV.get<Incident>(incident.id, 'json');
+    // Track if this is the first run, so we can ignore resolved incidents
+    const firstRun = await env.KV.get('firstRun', 'text') === null;
```

Comment:
no need to specify text, that's default and either way we don't mind how the content is delivered
```suggestion
    const firstRun = await env.KV.get('firstRun') === null;
```